src/seedu/addressbook/data/person/address/Address.java

```diff
@@ -56,4 +77,20 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    public AddressBlock getBlock() {
```

Comment:
Instead of sending each component separately, consider sending all of these values in the toString() method.